app/policies/episode_policy.rb

```diff
@@ -1,10 +1,11 @@
 class EpisodePolicy < ApplicationPolicy
   def create?
-    token && token.authorized?(account_id)
+    update?
   end
 
   def update?
-    token && token.authorized?(account_id)
+    token&.authorized?(account_id, :episode) ||
+      ( token&.authorized?(account_id, :episode_draft) && resource.draft? && resource.was_draft? )
```

Comment:
Does this imply a user with `episode-draft` can publish an episode ... but then cannot edit the episode after it's published (even if published_at is in the future)?

I see the case for preventing unpublishing - but why not let them edit it when it's published or released (with a future published_at)?  Mostly trying to remember the use cases around this...

Reply:
They can't publish an episode at all - they are not able to touch episodes that have a published_at date at all, whether in the future or not.

I think I was sort of thinking this would be useful as an approval-like flow, or maybe for apps that do imports but you actually have to hit OK on (though that might be overloading the published_at somewhat)